### PR TITLE
Added logfile truncation and download buttons to developer mdoe screen

### PR DESCRIPTION
**Pull request by jamez70**

### CODE REVIEW

**Comment by jamez70** [comment]
Updated style for button layout - please merge into trunk when you can

**Comment by jamez70** [comment]
Did you have a chance to look this over?

File: main/gen_gdl90.go [hunk 1]
```diff
@@ -1238,12 +1243,46 @@ func gracefulShutdown() {
...
 
+func reopenLogFile() {
+	// Duplicate log.* output to debugLog.
+	fp, err := os.OpenFile(debugLogf, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
+	if err != nil {
+		err_log := fmt.Errorf("Failed to open '%s': %s", debugLogf, err.Error())
+		addSystemError(err_log)
+		log.Printf("%s\n", err_log.Error())
+	} else {
+		defer fp.Close()
```
Comment (cyoung) [inline main/gen_gdl90.go]: Doesn't this close the logfile immediately (at the end of this function)?

File: main/gen_gdl90.go [hunk 2]
```diff
@@ -1238,12 +1243,46 @@ func gracefulShutdown() {
 	os.Exit(1)
 }
 
+func reopenLogFile() {
```
Comment (cyoung) [inline main/gen_gdl90.go]: Is this unused?

**Comment by jamez70** [comment]
I'll take a look this evening, but I know that is part of the developer
feature of clearing the log. I may have done it differently in the end, I
can't remember right now

**Pull request merged**